doc/source/aruba.md

```diff
@@ -0,0 +1,51 @@
+
+## Beaches
+
+Among the multiple beaches that you will find on the island is Eagle Beach, which is the widest in Aruba, where large hotels and water sports abound, in the same way you can find them in Druif Beach and Palm Beach. But if you prefer to visit quieter beaches you can go to the south of the island, specifically to Rodger's, a beach very popular with locals and fishermen, or to the north where Cura di Tortuga or Boca Catalina are located, where you will find two natural pools formed of rock volcanic and capable of protecting these beaches from the great waves of the sea.
+
+![Aruba](_static/images/aruba/beach.png)
+
+## Diving or snorkeling
+
+If instead of resting on white sand and bathing in turquoise waters you prefer to investigate the marine fauna, the perfect beaches for swimming, snorkeling or diving are Baby Beach and Arashi Beach. However, the recommendation for the most experienced is MangelHalto, which is protected by reefs where you can see yellow tail snapper, blue spike snapper, parrot fish and at deeper dive points, you can see barracudas, moray eels, octopus and other types of marine fauna.
+
+![Aruba](_static/images/aruba/diving.png)
+
+## To surf
+
+If you are a lover of challenging the waves, the recommendation is to go to Boca Grandi, which is located in the south of the island, yes, this beach is only suitable for experts, since the currents are usually excessively strong. For those people who prefer to observe, they can be entertained watching the jumps of the kitesurfers. For those who like bodyboarding, the ideal beach in this case is Andicuri, which is also the occasional venue for competitions of this type, being for many the best beach in the Caribbean to do this activity.
+
+## Tour Aruba on a Harley Davidson
+
+For motorcycle lovers, there is the possibility of exploring the island on a Harley Davidson, but they also offer you the possibility of touring it in a 4 × 4 vehicle, on horseback, bicycle, segway or quadricycle.
+
+## Ostrich Farm
+
+Visiting the Ostrich Farm can be a very interesting experience, since you will be able to learn about the behavior of these curious animals and you will have the opportunity to feed them, you can also see the Butterfly House that is located inside a tropical garden.
+Another interesting stop on the outskirts of the city is the Aloe Museum, located in the Hato Plantation. You can also live an intense adventure in the Arikok National Park located in the center of the island. There you can appreciate the dividivi trees, tropical flowers, exotic cacti, iguanas, lizards, birds, etc. In this park you will be able to observe a unique cultural heritage, since in the enclosure you will find caves with original rock art of the natives of the island, unusual geographic formations of limestone, quartz and lava. It even has walks through dry rivers that will take you to hidden bays like Boca Prins, Moro and Dos Playas.
```

Comment:
```suggestion

Another interesting stop on the outskirts of the city is the Aloe Museum, located in the Hato Plantation. You can also live an intense adventure in the Arikok National Park located in the center of the island. There you can appreciate the dividivi trees, tropical flowers, exotic cacti, iguanas, lizards, birds, etc. In this park you will be able to observe a unique cultural heritage, since in the enclosure you will find caves with original rock art of the natives of the island, unusual geographic formations of limestone, quartz and lava. It even has walks through dry rivers that will take you to hidden bays like Boca Prins, Moro and Dos Playas.
```